src/Model/Post.php

```diff
@@ -163,6 +163,7 @@ public function __construct( WP_Post $post ) {
 		}
 
 		$restricted_cap = $this->get_restricted_cap();
+		$restricted_cap = apply_filters( 'graphql_post_restricted_cap', $restricted_cap, $post, $this );
```

Comment:
What functionality are you looking to achieve here? We have the `graphql_restricted_data_cap` filter in the [abstract Model class](https://github.com/wp-graphql/wp-graphql/blob/8eb7dcf40cd30bd739d4d2411779fa7931f26cd2/src/Model/Model.php#L223) that all the other models extend, does that not work for your use case?

Reply:
Yeah, this works for me. I didn't see this hook, Thank you!